scalameta/semanticdb-integration/src/main/scala/example/Exclude.scala

```diff
@@ -0,0 +1,4 @@
+package exclude
+
+// This class should not be excluded by semanticdb.
```

Comment:
Should or should not? The build configuration says `s"-P:semanticdb:exclude:Exclude.scala",`

Reply:
Should be excluded, good catch!